### PR TITLE
Create gpu-1.7 job and add to 1.7-blocking dashboard

### PR DESCRIPTION
**Pull request by krzyzacy**
/assign @dchen1107 @vishh

### CODE REVIEW

File: jobs/ci-kubernetes-e2e-gce-gpu-1-7.env [hunk 1]
```diff
@@ -0,0 +1 @@
+PROJECT=k8s-jkns-e2e-gce-gpus-1-7
```
Comment (vishh) [inline jobs/ci-kubernetes-e2e-gce-gpu-1-7.env]: If this is a new GCE project, we will need GPU quota in `us-west1` region.

**Comment by dchen1107** [comment]
Please address the comments from @vishh 

/lgtm

File: jobs/ci-kubernetes-e2e-gce-gpu-1-7.env (hunk 1 above)
Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-gce-gpu-1-7.env]: Can you send a quota request? I'm not sure what will be needed. Thanks!

Comment (vishh) [inline jobs/ci-kubernetes-e2e-gce-gpu-1-7.env]: Submitted a quota request. This PR can go in. Once the request is approved I expect the tests to go green

**Pull request merged**